components/CreatedMoment/CreatedMoment.tsx

```diff
@@ -21,6 +56,24 @@ const CreatedMoment = () => {
           </div>
         </div>
       </div>
+      <TokenProvider
+        token={{
+          tokenContractAddress: createdContract as Address,
+          tokenId: createdTokenId,
+        }}
+        chainId={CHAIN_ID}
+      >
+        <CollectionProvider
+          collection={{
+            address: createdContract as Address,
+            chainId: CHAIN_ID,
+          }}
+        >
+          <AirdropProvider>
+            <Airdrop />
+          </AirdropProvider>
+        </CollectionProvider>
+      </TokenProvider>
```

Comment:
Open Closed Principle
- actual: new HTML code added inline to existing react component.
- required: new HTML component defined for new code. import the component into this existing code.